test/kubemark/start-kubemark.sh

```diff
@@ -320,6 +320,8 @@ current-context: kubemark-context")
   sed -i'' -e "s/{{registry}}/${CONTAINER_REGISTRY}/g" "${RESOURCE_DIRECTORY}/hollow-node.json"
   sed -i'' -e "s/{{project}}/${PROJECT}/g" "${RESOURCE_DIRECTORY}/hollow-node.json"
   sed -i'' -e "s/{{master_ip}}/${MASTER_IP}/g" "${RESOURCE_DIRECTORY}/hollow-node.json"
+  sed -i'' -e "s/{{kubelet_verbosity_level}}/${KUBELET_TEST_LOG_LEVEL}/g" "${RESOURCE_DIRECTORY}/hollow-node.json"
```

Comment:
That happens already by default (Ref: https://github.com/kubernetes/kubernetes/blob/master/cluster/kubemark/gce/config-default.sh#L61)